charts/rbac-definitions/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.rbacDefinitions }}
+Deployed RBACDefinitions:
+{{- range .Values.rbacDefinitions }}
```

Comment:
wouldn't this need to match what is happening in the definitions: 
```suggestion
{{- range .Values.rbacDefinitions }}
{{- $enabled := printf "%t" .enabled }}
{{- if regexMatch "nil" $enabled }}{{- $enabled = "true" }}{{- end }}
{{- if (eq $enabled "true") }}
```

Reply:
yeah you are right, it should match what is in the template, although I don't have any `enabled` value in the `Values.yaml` we need to fix this also in `cert-issuers` too then.

I'll apply this suggestion from you, and add an additional `test` to check it out.